src/data-types/wrappers.h

```diff
@@ -0,0 +1,326 @@
+#pragma once
```

Comment:
- Could you use #ifndef / #define / #endif instead?
- Rename the file to syscall_wrappers.h
- Could you add a global flag that would change the behavior of Fopen() to just call fopen() (and not do the loop) and same for other syscalls?